sbt

```diff
@@ -6,8 +6,8 @@
 
 set -o pipefail
 
-declare -r sbt_release_version="1.3.6"
-declare -r sbt_unreleased_version="1.3.6"
+declare -r sbt_release_version="1.3.7"
+declare -r sbt_unreleased_version="1.3.7"
```

Comment:
This declaration looks misleading. The same change was applied in #265 but it sounds to me like `sbt_unreleased_version`might need to point to `1.3.8`?

Reply:
`sbt_unreleased_version` is used for pre-releases, like release candidates, of which there aren't any for 1.3.8.